filehandling.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+
+struct Employee{ 
+	char employeeName[50]; 
+	int employeeId;
+};
+int main()
+{
+	FILE *fp = NULL;
+	int n;
+	printf("enter number of employee:"); 
+	scanf("%d", &n);
+	struct Employee emp[n];
+	int index;
+	for(index = 0; index < n; index ++){ 
+		printf("Enter the employee details\n"); 
+	        printf("Name:");
+                scanf("%s", emp[index].employeeName);
+                printf("Id:");
+                scanf("%d", &emp[index].employeeId);
+        }
+        fp=fopen("file1.bin", "wb");
+        if(fp == NULL){
+		printf("error in open file in write mode");
+        }
+        else{
+		for(index = 0; index < n; index ++){
+        	fwrite(&emp, sizeof(struct Employee), n, fp);
```

Comment:
Shouldn't this index to each employee? What's the use of `for` loop here otherwise

Reply:
What's the use of for loop here otherwise?
write the  each employee name and id into the file so I used  for loop here.

Reply:
For that shouldn't u use `&emp[index]` ?

Reply:
Just wondering how it worked. @Aswin31671  did u able to see expected output with this?

Reply:
now  I  changed in new pull request